.github/workflows/nodejs.yml

```diff
@@ -12,7 +12,11 @@ jobs:
 
     strategy:
       matrix:
-        node-version: [12.x, 14.x, 16.x]
+        node-version:
```

Comment:
Because the matrix includes `latest`, when `lts/*` begins pointing to a new version, no failures should occur, because that new version would have already been tested, back when it was `latest`. So non-fixed (_viz._ `lts/-2`, `lts/-1`, `lts/*`) values ought to be safe.

That said, when `latest` itself begins pointing to a new version, failures might occur. But if we remove `latest`, then `lts/*` loses the safety described above.

Reply:
> when `latest` itself begins pointing to a new version, failures might occur

Here’s one workaround: For `latest` (and only `latest`), if any step of the job fails ([`failure()`](https://docs.github.com/en/actions/learn-github-actions/expressions#failure)), [open an issue](https://github.com/github/accessibility-scorecard/blob/main/.github/actions/report-failure/action.yml), then `exit 0` so the overall workflow run is still green.

Reply:
Okay! cool. I got this working!

Here's a sample failed run: https://github.com/github/eslint-plugin-github/actions/runs/3884555154, and the issue it created: https://github.com/github/eslint-plugin-github/issues/378

To test this I created a failing test https://github.com/github/eslint-plugin-github/pull/369/commits/5077f3006ce588328af93f049cae1ad49b781220 which I will now delete.